packages/timeline-state-resolver/src/integrations/httpSend/index.ts

```diff
@@ -1,154 +1,76 @@
-		this._makeReadyDoesReset = initOptions.makeReadyDoesReset || false
-		this._resendTime = initOptions.resendTime && initOptions.resendTime > 1 ? initOptions.resendTime : undefined
-
-		return Promise.resolve(true) // This device doesn't have any initialization procedure
+	async init(options: DeviceOptions): Promise<boolean> {
+		this.options = options
+		this.cacheable = new CacheableLookup()
+		return true
 	}
-	/** Called by the Conductor a bit before a .handleState is called */
-	prepareForHandleState(newStateTime: number) {
-		// clear any queued commands later than this time:
-		this._doOnTime.clearQueueNowAndAfter(newStateTime)
-		this.cleanUpStates(0, newStateTime)
+	async terminate(): Promise<boolean> {
+		return true
 	}
-	handleState(newState: Timeline.TimelineState<TSRTimelineContent>, newMappings: Mappings) {
-		super.onHandleState(newState, newMappings)
-		// Handle this new state, at the point in time specified
-
-		const previousStateTime = Math.max(this.getCurrentTime(), newState.time)
-		const oldState: Timeline.TimelineState<TSRTimelineContent> = (
-			this.getStateBefore(previousStateTime) || { state: { time: 0, layers: {}, nextEvents: [] } }
-		).state
-
-		const convertTrace = startTrace(`device:convertState`, { deviceId: this.deviceId })
-		const oldHttpSendState = oldState
-		const newHttpSendState = this.convertStateToHttpSend(newState)
-		this.emit('timeTrace', endTrace(convertTrace))
-
-		const diffTrace = startTrace(`device:diffState`, { deviceId: this.deviceId })
-		const commandsToAchieveState: Array<any> = this._diffStates(oldHttpSendState, newHttpSendState)
-		this.emit('timeTrace', endTrace(diffTrace))
-
-		// clear any queued commands later than this time:
-		this._doOnTime.clearQueueNowAndAfter(previousStateTime)
-		// add the new commands to the queue:
-		this._addToQueue(commandsToAchieveState, newState.time)
 
-		// store the new state, for later use:
-		this.setState(newState, newState.time)
-	}
-	clearFuture(clearAfterTime: number) {
-		// Clear any scheduled commands after this time
-		this._doOnTime.clearQueueAfter(clearAfterTime)
-	}
-	async terminate() {
-		this._doOnTime.dispose()
-		return Promise.resolve(true)
+	get connected(): boolean {
+		return false
 	}
-	getStatus(): DeviceStatus {
-		// Good, since this device has no status, really
+	getStatus(): Omit<DeviceStatus, 'active'> {
 		return {
 			statusCode: StatusCode.GOOD,
 			messages: [],
-			active: this.isActive,
 		}
 	}
-	async makeReady(okToDestroyStuff?: boolean): Promise<void> {
-		if (okToDestroyStuff) {
-			if (this._makeReadyDoesReset) {
-				await this.resync()
-			}
 
-			for (const cmd of this._makeReadyCommands || []) {
-				await this.sendCommand(cmd)
-			}
-		}
+	actions: Record<string, (id: HttpSendActions, payload?: Record<string, any>) => Promise<ActionExecutionResult>> = {
+		[HttpSendActions.SendCommand]: async (_id: HttpSendActions.SendCommand, payload?: SendCommandPayload) =>
```

Comment:
Should the `resync` action also be implemented here, or is that handled elsewhere?